Vitals/ContentView.swift

```diff
@@ -114,7 +114,7 @@ struct ContentView: View {
                             .frame(width: 45, alignment: .trailing)
                     }
                 }
-            }.frame(maxWidth: .infinity, maxHeight: .infinity).padding(10)
+            }.frame(maxWidth: .infinity, maxHeight: .infinity).padding(EdgeInsets(top: 4, leading: 22, bottom: 4, trailing: 12))
```

Comment:
LGTM on Big Sur.
<img width="446" alt="Screenshot 2021-03-27 at 18 01 32" src="https://user-images.githubusercontent.com/18508536/112728222-85211880-8f26-11eb-854f-af3408c1b7d8.png">

Reply:
I noticed the process names aren't aligned with the other menu items on Big Sur (whereas they are on Catalina). I've tried to [account for this](https://github.com/hmarr/vitals/pull/11/commits/e70c18384309354ffac45800d3a41fe8c8342eb3#diff-5086d29a15216f17e942448f8ad96bbf073907532b24c1673750ff64608d9473R82), but I still don't have access to Big Sur, so the padding value is a guess that's likely to be a bit off. If you get a second, would you mind checking and seeing what looks right?

Reply:
12 px inset
<img width="506" alt="Screenshot 2021-07-18 at 18 39 34" src="https://user-images.githubusercontent.com/18508536/126075382-10f3e554-1f30-4091-87ec-96c3553e049c.png">

14 px inset
<img width="506" alt="Screenshot 2021-07-18 at 18 42 59" src="https://user-images.githubusercontent.com/18508536/126075383-d1f0d704-6212-4044-9fb5-952c81e7382d.png">

In my very scientific testing using a Preview.app selection box's edge to compare vertical alignment, I've found 14 px works a bit better than the current 12 px.

Reply:
Top science! 🧑‍🔬 

I've [updated it to 14px](https://github.com/hmarr/vitals/commit/3f901a460dafc34e53573951616885bb130a4d88) and released in [v0.5](https://github.com/hmarr/vitals/releases/tag/v0.5).